clang/lib/AST/DeclCXX.cpp

```diff
@@ -857,8 +881,13 @@ void CXXRecordDecl::addedMember(Decl *D) {
         data().HasDeclaredCopyAssignmentWithConstParam = true;
     }
 
-    if (Method->isMoveAssignmentOperator())
+    if (Method->isMoveAssignmentOperator()) {
       SMKind |= SMF_MoveAssignment;
+    }
```

Comment:
That's an unrelated change - and our style is to not have braces around single statements

Reply:
It's been fixed in the new commit.